corehq/apps/sso/utils/session_helpers.py

```diff
@@ -31,16 +31,33 @@ def _get_saml_user_data_property(request, prop_slug):
     return value
 
 
+def _get_display_name_from_session(request):
```

Comment:
Is it possible to remove the dependence on request from this and other methods. In particular, all of these functions are named: `_from_session` rather than `_from_request`, implying that the request isn't necessary. Beyond that, we don't really care that its a formal 'session' object either -- it seems all that is necessary is that it has a dictionary-like interface, so we can call `.get()` on it. It would be nice if we could reduce these down to functions where I could just pass in a dict with the properties I cared about when testing them.